strapi/strapi
# fix(content-releases): Add single type to releases

### What does it do?

Now you are able to add singleType entries to releases to publish/unpublish them.

### How to test it?

1. Create one Release
2. Go to one SingleType with Draft & Publish enabled
3. You should see the "Releases" box and there add the singleType to any release.
